hooks/boot/04-services.rb

```diff
@@ -0,0 +1,36 @@
+require 'open3'
+
+module ServicesHookContextExtension
+  ALL_POSSIBLE_SERVICES = [
```

Comment:
I already know qpid and friends is currently missing. Question is, do we wait for qpid to disappear or also list it here?